src/components/ReportWelcomeText.js

```diff
@@ -66,46 +77,66 @@ const ReportWelcomeText = (props) => {
-                        </Text>
-                        <Text>
-                            {isResctrictedRoom
-                                ? `${props.translate('reportActionsView.beginningOfChatHistoryRestrictedPartTwo')}`
-                                : `${props.translate('reportActionsView.beginningOfChatHistoryPrivatePartTwo')}`}
-                        </Text>
-                    </>
-                ) : (
-                    <>
-                        <Text>
-                            {props.translate('reportActionsView.beginningOfChatHistory')}
-                        </Text>
-                        {_.map(chatUsers, ({displayName, pronouns}, index) => (
-                            <Text key={displayName}>
-                                <Text style={[styles.textStrong]}>
-                                    {displayName}
-                                </Text>
-                                {!_.isEmpty(pronouns) && <Text>{` (${pronouns})`}</Text>}
-                                {(index === chatUsers.length - 1) && <Text>.</Text>}
-                                {(index === chatUsers.length - 2) && <Text>{` ${props.translate('common.and')} `}</Text>}
-                                {(index < chatUsers.length - 2) && <Text>, </Text>}
+            {isPolicyExpenseChat && (
+                <>
+                    <Text>
+                        {props.translate('reportActionsView.beginningOfChatHistoryPolicyExpenseChatPartOne')}
+                    </Text>
+                    <Text style={[styles.textStrong]}>
+                        {/* Use the policyExpenseChat owner's first name or their email if it's undefined or an empty string */}
+                        {lodashGet(props.personalDetails, [props.report.ownerEmail, 'firstName']) || props.report.ownerEmail}
+                    </Text>
+                    <Text>
+                        {props.translate('reportActionsView.beginningOfChatHistoryPolicyExpenseChatPartTwo')}
+                    </Text>
+                    <Text style={[styles.textStrong]}>
+                        {lodashGet(props.policies, [`${ONYXKEYS.COLLECTION.POLICY}${props.report.policyID}`, 'name'], 'Unknown Policy')}
+                    </Text>
+                    <Text>
+                        {props.translate('reportActionsView.beginningOfChatHistoryPolicyExpenseChatPartThree')}
+                    </Text>
+                </>
+            )}
+            {isChatRoom && (
+                <>
+                    <Text>
+                        {isResctrictedRoom
+                            ? `${props.translate('reportActionsView.beginningOfChatHistoryRestrictedPartOne')}`
+                            : `${props.translate('reportActionsView.beginningOfChatHistoryPrivatePartOne')}`}
+                    </Text>
+                    <Text style={[styles.textStrong]}>
+                        {lodashGet(chatUsers, '[0].displayName', '')}
```

Comment:
It doesn't look possible for this to be null based on this line so maybe we don't need the `lodashGet()`

https://github.com/Expensify/App/blob/ed4567243a570c7c9e1cc08c9ccbd49c5af7957d/src/components/ReportWelcomeText.js#L80

But thinking more on this... could we use `props.report.reportName` directly instead of having the ternary in `chatUsers`?